.golangci.yml

```diff
@@ -6,6 +6,8 @@ linters:
   - bodyclose
   - gofmt
   - goimports
+  - gci
```

Comment:
WDYT about enforcing import grouping? Something like stdlib, third-party, and github.com/projectcontour for groups?

Reply:
(ref. https://golangci-lint.run/usage/linters/#gci)

Reply:
yeah I'm down with that 👍🏽 

I guess we should add to the possible new `make format` to format imports with `gci` as well